src/main/java/com/github/manolo8/darkbot/utils/http/ThrowFunction.java

```diff
@@ -0,0 +1,6 @@
+package com.github.manolo8.darkbot.utils.http;
```

Comment:
I'd move this out of the specific `http` folder, the `ThrowingFunction` is a nice-to-have as generic